sbteclipse-plugin/src/sbt-test/sbteclipse/02-contents/project/Build.scala

```diff
@@ -14,7 +14,8 @@ object Build extends Build {
         "org.scala-lang" % "scala-compiler" % "2.10.2",
         "biz.aQute" % "bndlib" % "1.50.0"
       ),
-      retrieveManaged := true
+      retrieveManaged := true,
+      EclipseKeys.createSrc := EclipseCreateSrc.Default - EclipseCreateSrc.Managed
```

Comment:
Adding a sub project that uses this line would certainly be nice, but mainly I was just suggesting instead of changing this line or removing the failing lines in build.sbt to change them so that they pass.